Kitodo/src/main/java/org/kitodo/production/services/data/ImportService.java

```diff
@@ -310,9 +310,7 @@ private SchemaConverterInterface getSchemaConverter(DataRecord record) throws Un
         KitodoServiceLoader<SchemaConverterInterface> loader =
                 new KitodoServiceLoader<>(SchemaConverterInterface.class);
         List<SchemaConverterInterface> converterModules = loader.loadModules().stream()
-                .filter(c -> c.supportsSourceMetadataFormat(record.getMetadataFormat())
-                        && c.supportsSourceFileFormat(record.getFileFormat())
-                        && c.supportsTargetMetadataFormat(MetadataFormat.KITODO)
+                .filter(c -> c.supportsSourceFileFormat(record.getFileFormat())
```

Comment:
Please changing "c" to something to more meaningful variable name e.g. "converter".

Reply:
Done